README.md

```diff
@@ -39,7 +39,7 @@ The following module configurations can be provided in a data service designer s
 
 | Name                | Environment Variables | Type    | Description                |
 | ------------------- | --------------------- | ------- | -------------------------- |
-| MQTT Broker Address | MQTT_BROKER           | string  | eg: test.mosquitto.org     |
+| MQTT Broker Address | MQTT_BROKER           | string  | eg: mqtt://test.mosquitto.org (please notice that the protocol used in the module must be explicitely stated. Supported protocols: `mqtt://` or `ws://`)  |
```

Comment:
I don't think we support websocket here, nor should we. Could you check it it works with MQTTS and port 8883?

Reply:
removed